test/common/upstream/load_balancer_fuzz_base.h

```diff
@@ -56,10 +56,16 @@ class LoadBalancerFuzzBase {
   uint16_t port_ = 80;
   uint8_t num_priority_levels_ = 0;
 
+  enum class Locality {
+    ONE,
```

Comment:
Tiny nit: I think A, B, C make sense, since there is no ordering.

Reply:
This enum was deleted.